dotCMS/src/main/java/com/dotcms/util/user/LiferayUserTransformer.java

```diff
@@ -153,7 +156,11 @@ public User fromMap(final Map<String, Object> map) {
             if (null != map.get("delete_date")) {
                 user.setDeleteDate((java.util.Date) map.get("delete_date"));
             }
-
+            if (null != map.get("additional_info")) {
+                user.setAdditionalInfo(Try.of(() -> new ObjectMapper()
+                        .readValue((String) map.get("additional_info"), HashMap.class))
```

Comment:
Not sure if we should create a new Object Mapper every time
We have experimented issues with the same strategy on import and PP with XStream

Take a look to DotObjectMapperProvider may be

Reply:
You're right. Good catch!

Reply:
Done